docs/sql-data-sources-csv.md

```diff
@@ -248,5 +248,11 @@ Data source options of CSV can be set via:
     <td>Compression codec to use when saving to file. This can be one of the known case-insensitive shorten names (<code>none</code>, <code>bzip2</code>, <code>gzip</code>, <code>lz4</code>, <code>snappy</code> and <code>deflate</code>).</td>
     <td>write</td>
   </tr>
+  <tr>
+    <td><code>inferDateType</code></td>
+    <td>false</td>
+    <td>Infers all DateType format for the CSV. If this is not set, it uses the default value, <code>false</code>.</td>
```

Comment:
What do you mean by `all DateType format`? I think we should say about the `dateFormat` option here.